src/screens/setup/create/ConfirmPassphrase.js

```diff
@@ -67,88 +69,31 @@ class ConfirmPassphrase extends Component {
      * @param {String} action 
      * @param {Number} x 
      */
-    addTag(tagItem, action, x) {
-        const state = this.state;
-        if (action == "init") {
-            state.scrambledTags[x].selected = true;  
-            
-        } 
-    }
 
-    /* Pass in index and remove it out of whatever state -> just removed an item from any state, pass in state and index */
-    /**
-     * Helper method for the addTag method
-     * This method goes through the logic of transfering the selected tag to the "currentStateVariable"
-     * list.
-     * "currentStateVariable"  = "scrambledTags" | "selectedTags"
-     * @param {String} tagItem 
-     * @param {String} currentStateVariable 
-     * @param {Number} currenIndex 
-     */
-    swapTag(tagItem, currentStateVariable, currenIndex) {
+    addWord(wordItem, scrambledListIndex){
         const state = this.state;
-        if (currentStateVariable == "scrambledTags") {
-            state.scrambledTags.push(tagItem);
-            state.selectedTags.splice(currenIndex, 1);
-            this.setState(state)
-        } else if (currentStateVariable == "selectedTags") {
-            state.selectedTags.push(tagItem);
-            state.scrambledTags.splice(currenIndex, 1);
-            this.setState(state)
-        } 
+        let oldStates = state.scrambledTags
+        oldStates[scrambledListIndex].selected = true
+        this.setState({ scrambledTags: oldStates})
+        state.selectedTags.push({"wordItem":wordItem, "scrambledWordIndex": scrambledListIndex});
+        this.setState(state)
```

Comment:
Use setState to grab old state.... 
this.setState((prevState) => {
  return {quantity: prevState.quantity + 1};
});

https://stackoverflow.com/questions/43040721/how-to-update-a-nested-state-in-react